devices/wisdom.js

```diff
@@ -0,0 +1,21 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const extend = require('zigbee-herdsman-converters/lib/extend');
+const ota = require('zigbee-herdsman-converters/lib/ota');
+const tuya = require('zigbee-herdsman-converters/lib/tuya');
+const utils = require('zigbee-herdsman-converters/lib/utils');
+const globalStore = require('zigbee-herdsman-converters/lib/store');
+const e = exposes.presets;
+const ea = exposes.access;
+
+const definition = {
+    zigbeeModel: ['HK-DIM-SW'],
+    model: 'dmz250',
+    vendor: 'Wisdom',
```

Comment:
Looks this device is from [Profulux](https://profolux.com/product/led-dimmer-zigbee-3-0-5-250w/). Can you move it to `profulux.js`?

Reply:
@Koenkk is fine by me, the device identifies itself as Wisdom - so I actually think Profolux is simply a reseller?

Reply:
also, did you notice there is no profOlux yet (not profUlux), there is only profAlux which is yet another manufacturer

Reply:
Can you provide a link of where you bought the device?

Reply:
that was through https://www.ledlampaanbiedingen.nl/zigbee-smart-wifi-led-dimmer-0-250-watt-werkt-via.html but it seems they are both white-labeling the same hardware. Again, the device identifies itself as `Wisdom` which is why I picked it

Reply:
I see, then let's keep it as is.